dashboards-reports/test/jest.config.js

```diff
@@ -19,4 +19,5 @@ module.exports = {
     '\\.(css|less|sass|scss)$': '<rootDir>/test/__mocks__/styleMock.js',
     '\\.(gif|ttf|eot|svg)$': '<rootDir>/test/__mocks__/fileMock.js',
   },
+  testEnvironment: 'jsdom',
```

Comment:
what's this for?

Reply:
@joshuali925 This was added in sql so I added just in case but I am not sure if it is needed in reporting. https://github.com/opensearch-project/dashboards-visualizations/pull/56#discussion_r831315362

Reply:
Unfortunately it did not